### ISSUE TITLE
missing locale file for `zh-TW`

### ISSUE THREAD

**Issue by poga** [tootsuite/mastodon#3452]
After upgrading to v1.4.1, The server returns 500 if `DEFAULT_LOCALE` is set to `zh-TW`. Here's the log:

```
ActionView::Template::Error (Can't find locale_zh-TW.js in /home/mastodon/mastodon/public/packs/manifest.json.

f25c17b]     21:     = stylesheet_pack_tag 'common', media: 'all'
f25c17b]     22:     = stylesheet_pack_tag stylesheet_for_layout, media: 'all'
f25c17b]     23:     = javascript_pack_tag 'common', integrity: true, crossorigin: 'anonymous'
f25c17b]     24:     = javascript_pack_tag "locale_#{I18n.locale}", integrity: true, crossorigin: 'anonymous'
f25c17b]     25:     = csrf_meta_tags
f25c17b]     26:
f25c17b]     27:     = yield :header_tags
```

After some digging, I've noticed that there's no `zh-TW.json` in `app/javascript/mastodon/locales`, which might be the root cause of the issue?

* * * *

- [x] I searched or browsed the repo’s other issues to ensure this is not a duplicate.
- [x] This bug happens on a [tagged release](https://github.com/tootsuite/mastodon/releases) and not on `master` (If you're a user, don't worry about this).

**Comment by ykzts** [comment]
@poga I added `zh-TW.json`.

However, it is only fallback to English. A translator is required to make it zh-TW.

**Comment by poga** [comment]
@ykzts Thanks for the quick help! I've uploaded the files to my server and it's working now.

**Comment by caasi** [comment]
I am trying to translate some of those messages.

Any suggestions?

```
{
  "account.block": "封鎖 @{name}",
  "account.block_domain": "隱藏來自 {domain} 的一切",
  "account.disclaimer": "這使用者來自其他副本，實際數字可能更大。",
  "account.edit_profile": "編輯用戶資訊",
  "account.follow": "關注",
  "account.followers": "專注者",
  "account.follows": "正在關注",
  "account.follows_you": "關注你",
  "account.media": "媒體",
  "account.mention": "提到 @{name}",
  "account.mute": "消音 @{name}",
  "account.posts": "貼文",
  "account.report": "檢舉 @{name}",
  "account.requested": "正在等待許可",
  "account.unblock": "取消封鎖 @{name}",
  "account.unblock_domain": "不再隱藏 {domain}",
  "account.unfollow": "取消關注",
  "account.unmute": "不再消音 @{name}",
  "boost_modal.combo": "下次你可以按 {combo} 來跳過",
  "column.blocks": "封鎖的使用者",
  "column.community": "本地時間軸",
  "column.favourites": "最愛",
  "column.follow_requests": "關注請求",
  "column.home": "首頁",
  "column.mutes": "消音的使用者",
  "column.notifications": "通知",
  "column.public": "聯盟時間軸",
  "column_back_button.label": "上一頁",
  "column_subheading.navigation": "瀏覽",
  "column_subheading.settings": "設定",
  "compose_form.lock_disclaimer": "你的帳號沒有{locked}。任何人都可以關注你，看到發給關注者的貼文。",
  "compose_form.lock_disclaimer.lock": "上鎖",
  "compose_form.placeholder": "在想些什麼？",
  "compose_form.privacy_disclaimer": "Your post will be delivered to mentioned users on {domains}. Do you trust {domainsCount, plural, one {that server} other {those servers}}? Post privacy only works on Mastodon instances. If {domains} {domainsCount, plural, one {is not a Mastodon instance} other {are not Mastodon instances}}, there will be no indication that your post is not a public post, and it may be boosted or otherwise made visible to unintended recipients.",
  "compose_form.publish": "Toot",
  "compose_form.publish_loud": "{publish}！",
  "compose_form.sensitive": "將此媒體標為敏感",
  "compose_form.spoiler": "將訊息隱藏在警告訊息之後",
  "compose_form.spoiler_placeholder": "內容警告",
  "confirmation_modal.cancel": "取消",
  "confirmations.block.confirm": "封鎖",
  "confirmations.block.message": "你確定要封鎖 {name} ？",
  "confirmations.delete.confirm": "刪除",
  "confirmations.delete.message": "你確定要刪除這個狀態？",
  "confirmations.domain_block.confirm": "隱藏整個網域",
  "confirmations.domain_block.message": "你真的真的確定要封鎖整個 {domain} ？多數情況下，比較推薦封鎖或消音幾個特定目標就好。",
  "confirmations.mute.confirm": "消音",
  "confirmations.mute.message": "你確定要消音 {name} ？",
  "emoji_button.activity": "活動",
  "emoji_button.flags": "旗幟",
  "emoji_button.food": "食物與飲料",
  "emoji_button.label": "插入表情符號",
  "emoji_button.nature": "自然",
  "emoji_button.objects": "物件",
  "emoji_button.people": "人",
  "emoji_button.search": "搜尋...",
  "emoji_button.symbols": "符號",
  "emoji_button.travel": "旅遊與地點",
  "empty_column.community": "本地時間軸是空的。公開寫點什麼吧！",
  "empty_column.hashtag": "這個主題標籤下什麼都沒有。",
  "empty_column.home": "你還沒關注任何人。造訪{public}或利用搜尋功能找到其他用戶。",
  "empty_column.home.inactivity": "Your home feed is empty. If you have been inactive for a while, it will be regenerated for you soon.",
  "empty_column.home.public_timeline": "the public timeline",
  "empty_column.notifications": "You don't have any notifications yet. Interact with others to start the conversation.",
  "empty_column.public": "There is nothing here! Write something publicly, or manually follow users from other instances to fill it up",
  "follow_request.authorize": "Authorize",
  "follow_request.reject": "Reject",
  "getting_started.appsshort": "Apps",
  "getting_started.faq": "FAQ",
  "getting_started.heading": "Getting started",
  "getting_started.open_source_notice": "Mastodon is open source software. You can contribute or report issues on GitHub at {github}.",
  "getting_started.support": "{faq} • {userguide} • {apps}",
  "getting_started.userguide": "User Guide",
  "home.column_settings.advanced": "Advanced",
  "home.column_settings.basic": "Basic",
  "home.column_settings.filter_regex": "Filter out by regular expressions",
  "home.column_settings.show_reblogs": "Show boosts",
  "home.column_settings.show_replies": "Show replies",
  "home.settings": "Column settings",
  "lightbox.close": "Close",
  "loading_indicator.label": "Loading...",
  "media_gallery.toggle_visible": "Toggle visibility",
  "missing_indicator.label": "Not found",
  "navigation_bar.blocks": "Blocked users",
  "navigation_bar.community_timeline": "Local timeline",
  "navigation_bar.edit_profile": "Edit profile",
  "navigation_bar.favourites": "Favourites",
  "navigation_bar.follow_requests": "Follow requests",
  "navigation_bar.info": "Extended information",
  "navigation_bar.logout": "Logout",
  "navigation_bar.mutes": "Muted users",
  "navigation_bar.preferences": "Preferences",
  "navigation_bar.public_timeline": "Federated timeline",
  "notification.favourite": "{name} favourited your status",
  "notification.follow": "{name} followed you",
  "notification.mention": "{name} mentioned you",
  "notification.reblog": "{name} boosted your status",
  "notifications.clear": "Clear notifications",
  "notifications.clear_confirmation": "Are you sure you want to permanently clear all your notifications?",
  "notifications.column_settings.alert": "Desktop notifications",
  "notifications.column_settings.favourite": "Favourites:",
  "notifications.column_settings.follow": "New followers:",
  "notifications.column_settings.mention": "Mentions:",
  "notifications.column_settings.reblog": "Boosts:",
  "notifications.column_settings.show": "Show in column",
  "notifications.column_settings.sound": "Play sound",
  "notifications.settings": "Column settings",
  "onboarding.done": "Done",
  "onboarding.next": "Next",
  "onboarding.page_five.public_timelines": "The local timeline shows public posts from everyone on {domain}. The federated timeline shows public posts from everyone who people on {domain} follow. These are the Public Timelines, a great way to discover new people.",
  "onboarding.page_four.home": "The home timeline shows posts from people you follow.",
  "onboarding.page_four.notifications": "The notifications column shows when someone interacts with you.",
  "onboarding.page_one.federation": "Mastodon is a network of independent servers joining up to make one larger social network. We call these servers instances.",
  "onboarding.page_one.handle": "You are on {domain}, so your full handle is {handle}",
  "onboarding.page_one.welcome": "Welcome to Mastodon!",
  "onboarding.page_six.admin": "Your instance's admin is {admin}.",
  "onboarding.page_six.almost_done": "Almost done...",
  "onboarding.page_six.appetoot": "Bon Appetoot!",
  "onboarding.page_six.apps_available": "There are {apps} available for iOS, Android and other platforms.",
  "onboarding.page_six.github": "Mastodon is free open-source software. You can report bugs, request features, or contribute to the code on {github}.",
  "onboarding.page_six.guidelines": "community guidelines",
  "onboarding.page_six.read_guidelines": "Please read {domain}'s {guidelines}!",
  "onboarding.page_six.various_app": "mobile apps",
  "onboarding.page_three.profile": "Edit your profile to change your avatar, bio, and display name. There, you will also find other preferences.",
  "onboarding.page_three.search": "Use the search bar to find people and look at hashtags, such as {illustration} and {introductions}. To look for a person who is not on this instance, use their full handle.",
  "onboarding.page_two.compose": "Write posts from the compose column. You can upload images, change privacy settings, and add content warnings with the icons below.",
  "onboarding.skip": "Skip",
  "privacy.change": "Adjust status privacy",
  "privacy.direct.long": "Post to mentioned users only",
  "privacy.direct.short": "Direct",
  "privacy.private.long": "Post to followers only",
  "privacy.private.short": "Followers-only",
  "privacy.public.long": "Post to public timelines",
  "privacy.public.short": "Public",
  "privacy.unlisted.long": "Do not post to public timelines",
  "privacy.unlisted.short": "Unlisted",
  "reply_indicator.cancel": "Cancel",
  "report.heading": "New report",
  "report.placeholder": "Additional comments",
  "report.submit": "Submit",
  "report.target": "Reporting",
  "search.placeholder": "Search",
  "search_results.total": "{count, number} {count, plural, one {result} other {results}}",
  "status.cannot_reblog": "This post cannot be boosted",
  "status.delete": "Delete",
  "status.favourite": "Favourite",
  "status.load_more": "Load more",
  "status.media_hidden": "Media hidden",
  "status.mention": "Mention @{name}",
  "status.mute_conversation": "Mute conversation",
  "status.open": "Expand this status",
  "status.reblog": "Boost",
  "status.reblogged_by": "{name} boosted",
  "status.reply": "Reply",
  "status.replyAll": "Reply to thread",
  "status.report": "Report @{name}",
  "status.sensitive_toggle": "Click to view",
  "status.sensitive_warning": "Sensitive content",
  "status.show_less": "Show less",
  "status.show_more": "Show more",
  "status.unmute_conversation": "Unmute conversation",
  "tabs_bar.compose": "Compose",
  "tabs_bar.federated_timeline": "Federated",
  "tabs_bar.home": "Home",
  "tabs_bar.local_timeline": "Local",
  "tabs_bar.notifications": "Notifications",
  "upload_area.title": "Drag & drop to upload",
  "upload_button.label": "Add media",
  "upload_form.undo": "Undo",
  "upload_progress.label": "Uploading...",
  "video_player.expand": "Expand video",
  "video_player.toggle_sound": "Toggle sound",
  "video_player.toggle_visible": "Toggle visibility",
  "video_player.video_error": "Video could not be played"
}
```

**Comment by ykzts** [comment]
@caasi  please create new Pull Request.

**Comment by caasi** [comment]
@ykzts My fault. I will create a Pull Request later.

**Comment by LNDDYL** [comment]
"follow"不應該翻譯為"追隨"嗎？

**Comment by caasi** [comment]
@LNDDYL Twitter uses "關注" so I use the same word.

Let's discuss in #3480.
